src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,47 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T1, T2> {
+    T1 first;
```

Comment:
bad naming. T and K suggested or something like it. and need access modifiers, please